openedx_tagging/core/tagging/rest_api/v1/views.py

```diff
@@ -215,9 +240,6 @@ def export(self, request, **_kwargs) -> HttpResponse:
         Export a taxonomy.
         """
         taxonomy = self.get_object()
-        perm = "oel_tagging.export_taxonomy"
-        if not request.user.has_perm(perm, taxonomy):
-            raise PermissionDenied("You do not have permission to export this taxonomy.")
```

Comment:
It's true that "if you can view the taxonomy, you can export it", but are we still checking that the user has "view" permission here? I don't see it.

Reply:
The `get_object()` function handles that:

https://github.com/encode/django-rest-framework/blob/1db19f4b2d1500894448634852e02f73043382e4/rest_framework/generics.py#L103